.github/workflows/images.yml

```diff
@@ -10,6 +10,8 @@ env:
   GHCR_UI_IMAGE_NAME: ${{ github.repository }}/ui
   QUAY_REGISTRY: quay.io
   QUAY_UI_IMAGE_NAME: instructlab-ui/ui
+  GHCR_PATHSERVICE_IMAGE_NAME: ${{ github.repository }}/pathservice
```

Comment:
If I understand the functionality properly. This service provides people with the directory structure of the taxonomy so they can identify where there new contribution sits. I am not sure `pathservice` is the best description of this? taxonomy-descripion-service?

Reply:
Yes, it's basically provides you a list of first level directories within given directory in taxonomy repo. Which will be used by the pathservice ui component to provide the navigation feature. Naming is hard. This service helps user to find the path where they can place their files, hence the pathservice. Not sure if taxonomy-description-service fits the intent as well. But i am open to any name that makes sense.

Reply:
Yeh see what you mean. Maybe taxonomy-path-service?